aws/logs_monitoring/lambda_function.py

```diff
@@ -16,8 +16,11 @@
 import ssl
 import six.moves.urllib as urllib  # for for Python 2.7 urllib.unquote_plus
 import itertools
+
```

Comment:
Don't forget to bump up the version: https://github.com/DataDog/datadog-serverless-functions/pull/159/files#diff-618731ddf1c446ef45d09613cb4b189bR203